src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,102 @@
+.nav {
+  display: flex;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background-color: $colorNavback;
+  transition: transform 0.3s;
+  transform: rotateX(0);
+
```

Comment:
checklist 17: Fix menu for small screens (if there is not enough space for all the menu items

checklist 18: when the menu is open, the page can be scrolled using the tab button
![menu-small-screen](https://user-images.githubusercontent.com/89162624/209547418-e9139409-fa5b-4e2e-b70e-a81b9917e534.png)
![menu-tab-scrolling](https://user-images.githubusercontent.com/89162624/209547419-e86ae846-e7a5-437a-baeb-e779740c725f.png)

Reply:
thx, i fixed it